sdk/core/azure-core/inc/azure/core/http/curl_transport.hpp

```diff
@@ -122,6 +122,20 @@ namespace Azure { namespace Core { namespace Http {
      */
     std::string CAInfo;
 
+#if defined(AZ_PLATFORM_LINUX)
```

Comment:
`#include <azure/core/platform.hpp>` in this file explicitly.

Reply:
Done